packages/devtools_app/lib/src/app.dart

```diff
@@ -566,6 +567,10 @@ List<DevToolsScreen> get defaultScreens => <DevToolsScreen>[
         const VMDeveloperToolsScreen(),
         createController: () => VMDeveloperToolsController(),
       ),
+      DevToolsScreen<void>(
+        const ProviderScreen(),
+        createController: () {},
```

Comment:
The irony... the provider screen doesn't have a provider for a controller yet. :)